SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -340,6 +346,24 @@ void readTemp(){
     printTemperature(thermometer); //Call  
 }
 
+/*
+@Author Anthony Path
+Measure voltage of each cell of motor's battery and print it to serial. Works for both 1.5 V batteries and 1.2 V NIMHs
+*/
+void sendVoltage(){ 
+int value = analogRead(VOLT_PIN); 
+float voltage = value * (5.00 / 1023.00*3.13/8);  // Convert the analog reading (which goes from 0 - 1023) to a voltage (0 - 5V):
+ //Formula taken from here: https://www.arduino.cc/en/Tutorial/ReadAnalogVoltage
+ //Multiplied 3.13 because 47 and 100 K Ohm resistors lower voltage about 3.13 times, then divided by 8 to get approximate voltage per cell.
+ if (voltage<voltage_Now && ++voltageDelay>=1000){ //change global variable and send voltage to phone only when voltage drops
```

Comment:
Even during testing It wouldn't send the data any more often than once every 5 seconds.
But otherwise it *could* potentionally clog the socket which is what we want to avoid.